README.md

```diff
@@ -36,3 +36,157 @@ This is the shared library defining public classes that constitute the worker in
 ### worker-document-testing
 
 This contains implementations of the testing framework to allow for integration testing of the Document Worker. The project can be found in [worker-document-testing](worker-document-testing).
+
+## Workflow event handlers
+
+The below are the various event handlers from the workflow scripts.
```

Comment:
Our workflow is one way worker events can be used, they don't have to be used for workflow so it is inappropriate to refer to them as workflow scripts.